Python/Tests/Core.UI/WebProjectTests.cs

```diff
@@ -220,6 +220,7 @@ public class WebProjectTests {
             app.WaitForOutputWindowText("Build", "1 failed");
         }
 
+        /*
```

Comment:
the entire method is commented. does it mean that this code is not needed? if so, how about removing the lines than commenting out?

Reply:
None of the tests in this class are run because the "TestMethod" attribute has been commented out for all of them, including this function. I commented out this test because removing the older Python versions causes a syntax error on line 227. I can't figure out exactly what this test is doing and unsure about how to properly change it to the newer version. Whoever enables these tests can uncomment the function and fix the syntax error.

Reply:
then, can you add your comment above into this method header? it will let others know why this method is commented out. 
btw, you don't have to explicitly comment out "TestMethod" attribute since you already comment out the entire method with "/* */". 

---
In reply to: [292125876](https://github.com/microsoft/PTVS/pull/5363#discussion_r292125876) [](ancestors = 292125876)

Reply:
I added it to my commit yesterday. Do you see it?